lib/waterline/query/aggregate.js

```diff
@@ -51,8 +51,67 @@ module.exports = {
       return value !== undefined;
     });
     
+    // Validate each record in the array and if all are valid
+    // pass the array to the adapter's createEach method
+    var validateItem = function(item, next) {
+      _validate.call(self, item, next);
```

Comment:
Does this code do any extra validations in addition to `var errStr = _validateValues(_.cloneDeep(valuesList));` in [line 46](https://github.com/balderdashy/waterline/pull/1018/files#diff-762fb5e96b0c3cfe800c9025650adbb6R46)?